canonical-web-and-design/juju.is
# Make survey charts legible on mobile

## Done
Made the survey report charts more legible on mobile

## QA
- Go to https://juju-is-284.demos.haus/cloud-native-kubernetes-usage-report-2021
- Resize viewport to mobile width
- Compare charts against those on [the live page](https://juju.is/cloud-native-kubernetes-usage-report-2021)
- Check that they are more legible

## Issue
Fixes #282 

## Screenshots

### Before
<img width="327" alt="Screenshot 2021-06-25 at 10 17 45" src="https://user-images.githubusercontent.com/501889/123401488-c2855880-d59e-11eb-8731-de9ce606161a.png">

### After
<img width="327" alt="Screenshot 2021-06-25 at 10 17 55" src="https://user-images.githubusercontent.com/501889/123401514-cadd9380-d59e-11eb-9f57-7a7c9a23d6a5.png">
